src/components/VirtualizedAutocomplete/index.tsx

```diff
@@ -231,7 +232,7 @@ const VirtualizedAutocompleteBase = <
 
 	return (
 		<Autocomplete<Value, Multiple, DisableClearable, FreeSolo, ChipComponent>
-			{...props}
+			{...omit(props, 'loadNext')}
```

Comment:
How about just destrucuring it right above like `const { loadNext, ...passhroughProps } = props;`?
Lodash's Omit is one of the slow methods they offer, and given that it's just one prop, we can easily avoid it.